README.md

```diff
@@ -54,7 +54,7 @@ Options with `none` as the default are required.
 | `--repo` | `string` | `none` | `--repo osirrc2019/anserini` | the repo on Docker Hub
 | `--tag` | `string` | `latest` | `--tag latest` | the tag on Docker Hub
 | `--collections` | `[name]=[path]=[format] ...` | `none` | `--collections robust04=/path/to/robust04=trectext ...` | the collections to index
-| `--save_id` | `string` | `save` | `--save_id robust04-exp1` | used to calculate the ID for intermediate image after indexing
+| `--save_to_tag` | `string` | `save` | `--save_to_tag robust04-exp1` | used to calculate the ID for intermediate image after indexing
```

Comment:
Naming is hard... In retrospect I would personally hesitate to call it either `save_id` or `save_to_tag` because users might confuse it with an actual Docker image's id or tag, which this is of course not.

Reply:
Any suggestions? As long as the documentation is clear, I think it's fine with using either `id` or `tag` in the name instead of some contrived name.